### ISSUE TITLE
metadata doesn't match configurations files

### ISSUE THREAD

**Issue by CelesteHargreeves** [bmaltais/kohya_ss#3207]
No matter how I try to change the values, my learning_rate keeps being changed to "2e-06" in metadata. in kohya/config file i set the learning_rate to 1e-4. i have downloaded models from other creators on civitai and huggingface and their metadata always shows their intended learning_rate. I don't understand what is happening. I am training a flux style lora.

**Comment by b-fission** [comment]
learning_rate is probably being overridden by the separate "Unet learning rate" and "Text encoder learning rate" options. Try changing those.
